### ISSUE TITLE
dataset version 0.0

### ISSUE THREAD

**Issue by redorangeyellowy** [facebookresearch/InterHand2.6M#122]
Thank you for your excellent research.

According to one [paper](https://openaccess.thecvf.com/content/ICCV2021/papers/Kim_End-to-End_Detection_and_Pose_Estimation_of_Two_Interacting_Hands_ICCV_2021_paper.pdf), there seems to be a 0.0 version of the interhand dataset.

I think the dataset version you are currently deploying is 1.0.

Can you tell me the size of the image for each single-hand frame and interacting-hand frame of the 0.0 version dataset by train/val/test?

**Comment by mks0601** [comment]
This is v0.0 information. Please note that we do not release v0.0 now.

Train set
* Train (H): 76,445 single hand frames / 208,271 interacting hand frames / 284,716 total frames (full InterHand2.6M: 142,231 / 386,251 / 528,482)
* Train (M): 322,675 single hand frarmes / 174,383 interacting hand frames / 497,058 total frames (full InterHand2.6M: 594,189 / 314,848 / 909,037)
* Train (H+M): 371,800 single hand frames / 366,802 interacting hand frames / 738,602 total frames (full InterHand2.6M: 687,548 / 673,514 / 1,361,062)

Validation set
* Val (M): 113,370 single hand frames / 70,917 interacting hand frames / 184,287 total frames (full InterHand2.6M: 234,183 / 145,942 / 380,125)

Test set
* Test (H): 18,399 single hand frames / 48,323 interacting hand frames / 66,722 total frames (full InterHand2.6M: 33,665 / 87,908 / 121,573)
* Test (M): 179,593 single hand frames / 106,582 interacting hand frames / 286,175 total frames (full InterHand2.6M: 455,303 / 272,284 / 727,587)
* Test (H+M): 197,992 single hand frames / 154,905 interacting hand frames / 352,897 total frames (full InterHand2.6M: 488,968 / 360,192 / 849,160)

Total set
* InterHand2.6M (v0.0): 1,275,786 (full InterHand2.6M: 2,590,347)

**Comment by redorangeyellowy** [comment]
Thanks.
